### PR TITLE
docs: Audit logs

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR documents how to configure applications logs to stylus audit log file and how to download logs.

## Changed Pages

💻 [Audit Logs](https://deploy-preview-3564--docs-spectrocloud.netlify.app/clusters/edge/local-ui/host-management/audit-logs/)

## Jira Tickets

🎫 [PE-4732](https://spectrocloud.atlassian.net/browse/PE-4732)

## Backports

Can this PR be backported?

- [ ] No. 

[PE-4732]: https://spectrocloud.atlassian.net/browse/PE-4732?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+   Ensure your logs follow the RFC 5424 Syslog protocol, with the message in JSON format. The JSON object must contain
+   the following keys: `edgeHostId`, `contentMsg`, `action`, `actor`, `actorType`, `resourceId`, `resourceName`,
+   `resourceKind`. If your log entries do not follow this RFE 5424 Syslog protocol, or do not contain the necessary
+   keys, they will still be logged, but will not show up in Local UI.
+
+</TabItem>
+
+<TabItem value="Send Palette Logs to Another File">
+
+3. Populate the file with the following configuration. Replace `your-file` with the name of your log file.
```
Comment (Dr-N00B) [inline docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md:98]: @lennessyy , this should be point 4, 3 is already defined. Also point 2,3 and 4 are optional

Comment (lennessyy) [reply docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md:98]: Hi Arun, so these are two tabs I made for two different use cases. One is to send stylus logs to another file, and the other is to send app logs to the stylus log file. User can do either or both. I just didn't want to create two tasks that are basically identically save for the conf file.

Can you explain what you mean by point 2, 3 ,4 are optional? That leaves step 1 which is just to clone the repo.

File: docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+### Validate
+
+1. Log in to Local UI. For more information, refer to [Access Local UI Console](access-console.md).
+
+2. In the left **Main Menu**, click **Audit Log**. Confirm that the logs from your applications are being collected.
+
+## View and Download Log File from Local UI
+
+Log files are stored in the `/var/log/` folder. The active log file `stylus-audit.log` stores up to one year's worth of
+logs. When the one period is reached, or when the file size reaches 100 MB, the log file is rotated out and archived. If
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md:140]:
```suggestion
logs. After a year, or when the file size reaches 100 MB, the log file is rotated out and archived. If
```

File: docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+the log messages are configured to be picked up by Local UI, they will show up in Local UI. You can also download all
+log files in the `/var/log/` file from Local UI, including files that are older than one-year old, as well as files that
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md:142]:
```suggestion
log files in the `/var/log/` file from Local UI, including files that are older than a year, as well as files that
```

File: docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+were not configured to be displayed by Local UI.
+
+### Prerequisites
+
+- An active Edge host installed in the `airgap` installation mode.
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md:147]:
```suggestion
- An active Edge host installed in the `airgap` mode.
```

File: docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+### Procedure
+
+1. Log in to Local UI. For more information, refer to [Access Local UI Console](access-console.md).
+
+2. In the left **Main Menu**, click **Diagnostics**. Then click the **Logs** tab. You can view all audit log entries on
+   this page. Local UI will display log entries dated up to one year ago.
+
+3. To download the log files, click the **Download** button in the upper-right corner. This will download all files in
+   the `/var/log` folder, including archived old log files as well as any additional log files you configured to be
```
Comment (yuliiiah) [inline docs/docs-content/clusters/edge/local-ui/host-management/audit-logs.md:157]:
```suggestion
   the `/var/log` folder, including the archived log files as well as any additional log files you configured to be
```

**Pull request merged**